src/Sddl.Parser/Ace.cs

```diff
@@ -407,5 +407,39 @@ public override string ToString()
 
             return sb.ToString();
         }
+
+        public static bool operator== (Ace ace0, Ace ace1)
+        {
+            bool isTypeEqual, areFlagsEqual, areRightsEqual, isObjectGuidEqual, isInheritObjectGuid;
+
+            isTypeEqual = ace0.AceType == ace1.AceType;
```

Comment:
@demonoidv thank you for your contribution :) I would suggest to write it in a way to chain equality comparisons of properties with logical and (&&), what short cicuts comparison after first inequality encountered. That approach will be a bit more performing. Same can be achieved with list of if-returns blocks. You can also generate equality operators with newer wersion of full VS or R#.

Also - for overall consistency - it may be reasonable to override equality operators for all domain object in repo.

However, if this this specific implementation is urgently required by you I may skip above suggestions and merge your proposal (since the repo is on early stage).

Reply:
@zacateras You're welcome, thank you for taking the time to review this :)
That's a good idea I will rewrite this.

Do you think I can override equality operators for the other objects in this PR or do you prefer a dedicated PR for each object ?

Don't worry it's not so urgent, it can wait some additionnal modifications 😄

Reply:
single PR will do the job. thanks!